help/CelestiaGuide.html

```diff
@@ -170,7 +170,7 @@ <h2>Getting Started Topics</h2>
 		The <i>Tour Guide</i> is a list of some of the more interesting objects you can visit in Celestia. Select the <i>Tour Guide</i> option in the <i>Navigation menu</i> to display the Tour Guide window. Choose a destination from the list, click the <b>Goto</b> button, and you're off.
 		<br>
 		<br>
-		That covers the very basics. For a more in-depth look at Celestia and the controls available to you, download the "Celestia User's Guide" (written by Frank Gregorio), available in several languages, from: <a href="http://www.shatters.net/celestia/documentation.html">http://www.shatters.net/celestia/documentation.html</a>. This web page also includes links to the Celestia README file translated into Japanese.
+		That covers the very basics. For a more in-depth look at Celestia and the controls available to you, download the "Celestia User's Guide" (written by Frank Gregorio), available in several languages, from: <a href="https://celestia.space/guides.html">https://celestia.space/guides.html</a>. This web page also includes links to the Celestia README file translated into Japanese.

```

Comment:
please remove "This web page also includes links to the Celestia README file translated into Japanese."

Reply:
Done, but I wonder why that translation is no longer there?

Reply:
Ask Alexell :)